examples/hosts/iframe-host/src/inner.ts

```diff
@@ -21,27 +27,23 @@ async function getFluidObjectAndRender(baseHost: BaseHost, url: string, div: HTM
 export async function runInner(divId: string) {
     const div = document.getElementById(divId) as HTMLDivElement;
 
-    const pkg: IFluidCodeDetails = {
-        package: "@fluid-example/todo@^0.15.0",
-        config: {
-            "@fluid-example:cdn": "https://pragueauspkn.azureedge.net",
-        },
-    };
-
     const documentServiceFactory = await InnerDocumentServiceFactory.create();
-    const baseHost = new BaseHost(
-        {
-            codeResolver: new SemVerCdnCodeResolver(),
-            documentServiceFactory,
-            urlResolver: documentServiceFactory.urlResolver,
-            options: {},
-        });
+    const module = { fluidExport: TodoContainer };
+    const codeLoader = { load: async () => module };
+
+    const loader = new Loader({
+        urlResolver: documentServiceFactory.urlResolver,
+        documentServiceFactory,
+        codeLoader,
+    });
 
     const url = documentServiceFactory.resolvedUrl.url;
-    const container = await baseHost.initializeContainer(url, pkg);
+    const documentId = decodeURIComponent(url.replace("fluid://localhost:3000/tinylicious/", ""));
```

Comment:
> const documentId = decodeURIComponent(url.replace("fluid://localhost:3000/tinylicious/", "")); [](start = 3, length = 95)

can't think come from the resolve url above?

Reply:
Nothing in `IFluidResolvedUrl` equal to this (copied this from the insecuretinyliciousurlresolver 🤔)